pkg/filter/regex.go

```diff
@@ -15,12 +15,12 @@ type RegexpFilter struct {
 func NewRegexpFilter(value string) (ffuf.FilterProvider, error) {
 	re, err := regexp.Compile(value)
 	if err != nil {
-		return &RegexpFilter{}, fmt.Errorf("Size filter or matcher (-fs / -ms): invalid value: %s", value)
+		return &RegexpFilter{}, fmt.Errorf("Regexp filter or matcher (-fr / -mr): invalid value: %s", value)
 	}
 	return &RegexpFilter{Value: re, valueRaw: value}, nil
 }
 
-func (f *RegexpFilter) Filter(response *ffuf.Response) (bool, error) {
+func (f *RegexpFilter) Filter(response *ffuf.Response, inputs map[string][]byte) (bool, error) {
```

Comment:
You can access the fuzz keywords through `response.Request.Input` and can remove the code passing the input data through the application.

Reply:
Ah, missed that, thx.